inference-engine/thirdparty/clDNN/src/gpu/device_info.cpp

```diff
@@ -30,6 +30,50 @@
 namespace cldnn {
 namespace gpu {
 
+namespace {
+
+bool is_local_block_io_supported(const cl::Device& device) {
```

Comment:
Why do you use this approach instead of simple extension check? 
```
extensions.find("cl_intel_subgroup_local_block_io") != std::string::npos;
```

Reply:
I'm not sure if extension can be already extracted from API, I'll check.
Also I wanted this to be more future-proof in case extension name changes, as it is not yet published.

Reply:
clinfo already shows it in the device extensions list for 20.20.16837 driver.

Reply:
Ok, I'll change it then